packages/EasyCore/src/Bridge/Symfony/ApiPlatform/Normalizers/CarbonImmutableNormalizer.php

```diff
@@ -18,7 +18,7 @@ public function denormalize(
         string $type,
         ?string $format = null,
         ?array $context = null
-    ): ?CarbonImmutable {
+    ): CarbonImmutable {
```

Comment:
Will it work with Symfony 5.4 and 6.0 at the same time?

Reply:
It works well with both versions